services/ai_service.py

```diff
@@ -0,0 +1,23 @@
+from abc import abstractmethod
+
+class AIService:
+    def __init__(self):
+        pass
+
+    def set_image_style(self, image_style):
+        self.image_style = image_style
+
+    # Speak the sentence. Returns None
+    @abstractmethod
+    def run_tts(self, sentence):
+        pass
+
+    # Generate an image from the sentence. Returns an image
```

Comment:
Same as above, it would be more useful for the caller and reader to add a type hint for the return type.